e2e/mock-api/project.json

```diff
@@ -1,17 +1,24 @@
 {
+  "name": "mock-api",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "e2e/mock-api/src",
   "projectType": "application",
   "targets": {
     "build": {
-      "executor": "@nrwl/node:webpack",
+      "executor": "@nrwl/esbuild:esbuild",
```

Comment:
🐎 faster / and newer / i was hitting a weird issue with webpack 5 (not weird but wasn't worth solving)